notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<ul><li>This function name is confusing. It seems it crate labelled images. Change it to "create_label_embedding" or something describing more precisely what's happening.</li><li>This code would be much more readable without the <code>tf.keras.layers</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Thanks. I modified the name and removed `tf.keras.layers` from this lab.